include/RAJA/policy/sycl/reduce.hpp

```diff
@@ -473,15 +609,57 @@ class ReduceMinLoc<sycl_reduce, T, IndexType>
   //! enable minloc() for ReduceMinLoc -- alias for reduce()
   self &minloc(T rhsVal, IndexType rhsLoc)
   {
+#ifdef __SYCL_DEVICE_ONLY__
+    // TODO: Race condition currently
```

Comment:
Why do minloc and maxloc produce race conditions here?

Reply:
Currently the location write can be disconnected with the min/max value write.